setup/partner_tz/odoo/addons/partner_tz

```diff
@@ -0,0 +1 @@
+../../../../partner_tz
```

Comment:
this work in every project scaffold ?

Reply:
IIRC these setup files are regenerated after merges by the OCA git bot...